subsys/bluetooth/host/l2cap.c

```diff
@@ -981,6 +981,16 @@ static uint16_t l2cap_chan_accept(struct bt_conn *conn,
 	return BT_L2CAP_LE_SUCCESS;
 }
 
+static bool l2cap_check_security(struct bt_conn *conn,
+				 struct bt_l2cap_server *server)
+{
+	if (IS_ENABLED(CONFIG_BT_CONN_DISABLE_SECURITY)) {
+		return true;
+	}
+
+	return (conn->sec_level < server->sec_level) ? false : true;
```

Comment:
This looks unnecessarily hard to read. Just reverse the condition instead of using a conditional to reverse the result.

Reply:
Done.